scripts/qa_crowbarsetup.sh

```diff
@@ -2326,7 +2326,11 @@ function custom_configuration()
         ceilometer)
             if [[ $hacloud = 1 ]] ; then
                 proposal_set_value ceilometer default "['deployment']['ceilometer']['elements']['ceilometer-server']" "['cluster:$clusternameservices']"
-                proposal_set_value ceilometer default "['deployment']['ceilometer']['elements']['ceilometer-cagent']" "['cluster:$clusternameservices']"
+                if iscloudver 6plus; then
+                    proposal_set_value ceilometer default "['deployment']['ceilometer']['elements']['ceilometer-polling']" "['cluster:$clusternameservices']"
+                else
+                    proposal_set_value ceilometer default "['deployment']['ceilometer']['elements']['ceilometer-cagent']" "['cluster:$clusternameservices']"
```

Comment:
@bmwiedemann: I added the variable. 
@vmoravec: If the scenario files are intended only for cloud6+, Can I remove 'ceilometer-cagent' and retain 'ceilometer-polling' because ceilometer-cagent is deprecated after Kilo release?